wp-content/themes/core/assets/js/utils/dom/apply-browser-classes.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Curious: Do you think there's a good reason to keep these browser-specific tests/classes around anymore? I can't recall the last time I utilized them for a project.

Reply:
I've had to use them once in the last year, so I figured having the basics set up would be nice. It's getting to be less and less though, I'd be fine taking it out and saving those extra 1-2 KBs of the JS size. Before we just remove it, thoughts on asking in the all-devs channel how often it gets used to get wider input?

Reply:
> Before we just remove it, thoughts on asking in the all-devs channel how often it gets used to get wider input?
Sounds good to me. Are you able to spin that up this week? (I'm ooo Weds  thru Mon next week.)

Reply:
Yea, for sure, I'll get a poll up tomorrow